exoscale/provider.go

```diff
@@ -142,6 +142,8 @@ func Provider() terraform.ResourceProvider {
 			"exoscale_ipaddress":            resourceIPAddress(),
 			"exoscale_network":              resourceNetwork(),
 			"exoscale_nic":                  resourceNIC(),
+			"exoscale_nlb":                  resourceNlb(),
```

Comment:
Same as "NIC" and "IP", "NLB" is an acronym therefore it must be capitalized:

* `resourceNLB`
* `resourceNLBService`

(applies everywhere in the code)

Reply:
fixed: 77d6edd